scripts/lint-changed-files.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Should we also add references to this to the Makefile, so developers can run it locally?

Reply:
Not sure what you mean here?